lib/glimesh/community_team/audit_log.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Glimesh.CommunityTeam.AuditLog do
+  @moduledoc false
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "gct_audit_log" do
+    belongs_to :user, Glimesh.Accounts.User
+
+    field :action, :string
+    field :target, :string
+    field :verbose_required?, :boolean
```

Comment:
Does this actually put a question mark in the SQL column name?

Reply:
![image](https://user-images.githubusercontent.com/11252574/99100531-ee247700-25a9-11eb-8452-175347bcfdec.png)
Yes :)